ydb/docs/en/core/concepts/datamodel/view.md

```diff
@@ -0,0 +1,84 @@
+# View
+
+A view is a query which is treated as if it was a table storing the results of the query. The view itself contains no data. The content of a view is generated every time you select from that view. Any changes in the underlying tables are reflected immediately in the view.
```

Comment:
I'm hesitant about the first sentence suggested change. What I tried to say in this sentence is that a view is almost like a table. It is important that the view itself "is treated as if it was a table", not that somewhere exists a table that stores the results of a query (which is a different way of saying "there was a table storing the results of a query").

I wouldn't like to change the first sentence.

Reply:
I wouldn't like to change the second sentence either, because "there's no caching involved" is a bit misleading. Caching can mean a lot of things and I think I have elaborated enough on the exact meaning of the "view itself contains no data" by providing a third sentence, which explains precisely what I mean in the second one.

Reply:
To be frank, I don't want to make the suggested change in the third sentence either. I don't want to mix different topics in one sentence here. "The content of a view is generated every time you `SELECT` from it" continues the second sentence and is elaborated a bit more in the fourth sentence, while "modifying queries is forbidden" is completely a different topic, which I haven't discussed anywhere in the article.

Reply:
Changed to:

A view is a query which is treated as if it was a table storing the results of a given query. The view itself contains no data. The content of a view is generated every time you `SELECT` from it. Thus, any changes in the underlying tables are reflected immediately in the view.